src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -5348,8 +5515,11 @@ def _make_onedeploy_request(params):
 
     # check if an error occured during deployment
     if response.status_code:
-        raise CLIError("An error occured during deployment. Status Code: {}, Details: {}"
-                       .format(response.status_code, response.text))
+        scm_url = _get_scm_url(params.cmd, params.resource_group_name, params.webapp_name, params.slot)
+        latest_deployment_url = scm_url + "/api/deployments/latest"
```

Comment:
latest --> deployment id

Reply:
Here there is a possibility that we might not have a deploymentid since kudu build failed and deploymentid will be a temp id. Incase we don't have the deployment id, redirecting to /latest